src/main.rs

```diff
@@ -41,5 +47,9 @@ fn main() {
                 },
             );
         }
+        Commands::Completion(arguments) => {
+            let mut cmd = App::command();
+            print_completions(arguments.shell, &mut cmd);
```

Comment:
This should return from an `exit` call like `encode` and `decode` above for proper exit status support. Since `generate` doesn't return a value, we can always exit `0` here:

```suggestion
            print_completions(arguments.shell, &mut cmd);
            exit(0)
```